Makefile

```diff
@@ -1,3 +1,8 @@
+BASE_PATH = $(shell pwd)
+BIN_PATH := $(BASE_PATH)/bin
+
+export PATH := $(BIN_PATH):$(PATH)
+
```

Comment:
I'm still not sure that we need this.

Reply:
It's already in other repositories.
If we want really get rid of it, let's discuss it again on Go sync. But for now I would like to merge it.

Reply:
Already discussed it with @oter. Yep, there is some benefits from this, so let's keep it.